core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -286,10 +286,11 @@ class KafkaZkClient private (zooKeeperClient: ZooKeeperClient, isSecure: Boolean
    * @throws KeeperException if there is an error while setting or creating the znode
    */
   def createConfigChangeNotification(sanitizedEntityPath: String): Unit = {
+    makeSurePersistentPathExists(ConfigEntityChangeNotificationZNode.path)
```

Comment:
I think this makes sense for 1.1 and to get the tests passing again, but maybe calling `zkClient.createTopLevelPaths()` from the config tool is a better long term solution. We can do the latter in a separate PR (I can include it in a related clean-up PR that I have been working on for too long).

Reply:
Sounds good to me. I considered doing just that in fact, but decided for the incremental fix instead.